### ISSUE TITLE
shortcut for laptop layout to announce current folder in folder list

### ISSUE THREAD

**Issue by burmancomp** [RPTools-org/thunderbirdPlusG5#4]
Currently selected folder is announced with nvda+up arrow. It should be possible to announce it also with nvda+l in laptop layout.
